assignments/arrays.js

```diff
@@ -63,32 +63,56 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+// console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+let carModels = [];
+for (let i = 0; i < inventory.length; i++)
+    carModels.push(inventory[i].car_model);
+console.log(carModels.sort());
+
+let carMakes = [];
+for (let i = 0; i < inventory.length; i++)
+    carMakes.push(inventory[i].car_make);
+console.log(carMakes.sort());
+
+let carYears = [];
+for (let i = 0; i < inventory.length; i++)
+    carYears.push(inventory[i].car_year);
+console.log(carYears.sort());
+
+console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}`);
+
+
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
```

Comment:
again here we know the position of the last car so we could do this
let lastCar = inventory[inventory.length-1];
console.log(lastCar.car_make, lastCar.car_model)